src/iuse.cpp

```diff
@@ -5227,6 +5227,20 @@ cata::optional<int> iuse::hotplate( Character *p, item *it, bool, const tripoint
     return cata::nullopt;
 }
 
+cata::optional<int> iuse::hotplate_atomic( Character *p, item *it, bool, const tripoint & )
+{
+    if( p->is_mounted() ) {
+        p->add_msg_if_player( m_info, _( "You can't do that while mounted." ) );
+        return cata::nullopt;
+    }
+    
+    if ( it->typeId() == itype_atomic_coffeepot ) {
```

Comment:
```suggestion
    if( it->typeId() == itype_atomic_coffeepot ) {
```

Here is the bit that astyle doesn't like - just remove this space between `if` and `(` and it should work.